src/client/Microsoft.Identity.Client/ApiConfig/BrokerOptions.cs

```diff
@@ -29,6 +29,10 @@ public enum OperatingSystems
             /// Use broker on Windows OS
             /// </summary>
             Windows = 0b_0000_0001,  // 1
+            /// <summary>
+            /// Use broker on Mac OS
+            /// </summary>
+            Mac = 0b_0000_0010,  // 10
```

Comment:
nit: consider naming this as `OSX` as this is what the .NET team does in https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform?view=net-9.0